src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,212 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
+$toggle-switch-handle: $iui-icons-default;
```

Comment:
Can you make it more specific? Because it is completely unclear to me by reading its name for what this variable is used.

Reply:
Changed to `$toggle-switch-handle-size`.